contents/blog/google-analytics-cost.md

```diff
@@ -0,0 +1,127 @@
+---
+title: How much does Google Analytics actually cost?
+date: 2024-11-28
+author:
+ - ian-vanagas
+rootpage: /blog
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-vs-ga4/posthog-vs-ga4.jpeg
+featuredImageType: full
+tags:
+ - Guides
+---
+
+Some of humanity's great mysteries: What happens after death? Where did life come from? Are aliens real? How much does [Google Analytics](/blog/google-analytics-to-posthog) actually cost? This post attempts to answer the only one that can be wrapped up in a single blog post. 
+
+Thanks to its connection with Google, early entrance into the [web analytics](/web-analytics) market, and initial price tag (free), Google Analytics has long been the most popular product in its category. According to [BuiltWith](https://trends.builtwith.com/analytics/Google-Analytics), as of November 2024, it's used by 50% of the top million sites.
+
+![BuiltWith graph](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_11_29_at_10_42_08_d308dcfae1.png)
+
+Although Google Analytics is largely seen as a free product, this doesn't tell the full story. This post attempts to tell the rest of the story and help you figure out how much Google Analytics actually costs. 
+
+## Google Analytics 4 is free (but limited)
```

Comment:
This heading is misleading IMO, since it does have a cost at some point. Would rephrase to something like this (not quite sure what the right one is)
```suggestion
## Google Analytics's free tier 
```

Reply:
Oh I get it now, GA4 and GA360 are two different things. I would talk about this first

Reply:
I clarified they are different in the opening section so this makes more sense